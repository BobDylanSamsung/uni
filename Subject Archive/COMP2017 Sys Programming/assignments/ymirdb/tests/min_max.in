set a 1 2 3 4 5 6
set b 3 2 5 4 1 6
set c -1 9 3 0 4
min a
min b
min c
max a
max b
max c
pop a
pluck a 5
pluck b 5
pluck b 5
pop c
pop c
list entries
min a
min b
min c
max a
max b
max c
pop a
list entries
bye
