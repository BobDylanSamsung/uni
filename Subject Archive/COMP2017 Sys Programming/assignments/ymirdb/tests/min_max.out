ok

ok

ok

1

1

-1

6

6

9

1

6

1

6

-1

9

c [3 0 4]
b [3 2 5 4]
a [2 3 4 5]

2

2

0

5

5

4

2

c [3 0 4]
b [3 2 5 4]
a [3 4 5]

bye
